spec/acceptance/bundle_management_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper_acceptance'
+require 'fileutils'
+
+describe 'Managing Gemfile dependencies' do
+  before(:all) do
+    # Do everything in a clean tmpdir.
+    @old_pwd = Dir.pwd
+    @tmpdir = Dir.mktmpdir('pdk-acceptance')
+
+    Dir.chdir(@tmpdir)
```

Comment:
Oh yes, I originally had the tmpdir code in spec_helper_acceptance and wrapped in a conditional for testmode == local. It would definitely be better to have the functionality built into the tooling.